helpers/integration.go

```diff
@@ -325,6 +326,21 @@ func HasPipelineRunSucceeded(object client.Object) bool {
 	return false
 }
 
+// GetPipelineWithName returns a PipelineRun based on the name and namespace
+// If PipelineRun is not found, a non-nil error is returned
+func GetPipelineWithName(pipelineRunName string, namespace string, k8sClient client.Client, ctx context.Context) (*tektonv1beta1.PipelineRun, error) {
```

Comment:
It looks like this function is not used, can we remove it?

Reply:
Thanks! removed it